apps/forms-flow-ai/forms-flow-web/src/components/FOI/ReviewRequest/AdditionalApplicantDetails.js

```diff
@@ -5,40 +5,45 @@ import TextField from '@material-ui/core/TextField';
 import './additionalapplicantdetails.scss';
 import { formatDate } from "../../../helper/FOI/helper";
 
+import FOI_COMPONENT_CONSTANTS from '../../../constants/FOI/foiComponentConstants';
 
-
-const AdditionalApplicantDetails = React.memo(({additionalInfo}) => {
+const AdditionalApplicantDetails = React.memo(({additionalInfo, createSaveRequestObject}) => {
     /**
      *  Addition Applicant details box in the UI
      *  No mandatory fields here
      */ 
   
     //local state management for personalHealthNumber, identityVerified, correctionNumber, and birthDate
-    const [personalHealthNumberText, setPersonalHealthNumber] = React.useState(!!additionalInfo.personalHealthNumber ? additionalInfo.personalHealthNumber : "0000 000 00");
-    const [identityVerifiedText, setIdentityVerified] = React.useState(!!additionalInfo.identityVerified ? additionalInfo.identityVerified : "");  
-    const [correctionsNumberText, setCorrectionsNumber] = React.useState(!!additionalInfo.correctionsNumber ? additionalInfo.correctionsNumber : "0000 000 00");
-    const [employeeNumberText, setEmployeeNumber] = React.useState(!!additionalInfo.employeeNumber ? additionalInfo.employeeNumber : "0000 000 00");
-    const dob = !!additionalInfo.birthDate ? new Date(additionalInfo.birthDate) : "";   
+    const [personalHealthNumberText, setPersonalHealthNumber] = React.useState(additionalInfo !== undefined && !!additionalInfo.personalHealthNumber ? additionalInfo.personalHealthNumber : "0000 000 00");
```

Comment:
I'd suggest writing a validation function here for the first part, as I can see this logic being used in more and more places.